schema.js

```diff
@@ -0,0 +1,23 @@
+const summarySchema = new mongoose.Schema({
+    restaurant_id: {type: Number, unique: true},
+    restaurant_name: String,
+    restaurant_rating: String,
+    restaurant_reviewCount: String,
+    restaurant_priceRange: String,
+    restaurant_category: String,
+    restaurant_description: String,
+    restaurant_tag: String
```

Comment:
@cminho921 
@LittleJennie 

There can be multiple tags, right? Will this be a single string, or multiple?